src/main/java/com/jw/entity/Order.java

```diff
@@ -26,6 +26,7 @@ public class Order {
 
     @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
     @JoinColumn(name = "orderid")
+    //REVIEW-VINI: Why do you need this @OnDelete? Does Cascade and orphanRemoval not enough?
```

Comment:
Without it deleteById seems to work fine, but deleteAll fails with database foreign key constraint validation
`Detail: Key (orderid)=(1) is still referenced from table "order_product_table"`